config/config.exs

```diff
@@ -17,7 +17,8 @@ config :nautic_net_device,
   udp_endpoint: System.fetch_env!("UDP_ENDPOINT"),
   product: System.fetch_env!("PRODUCT"),
   replay_log: System.get_env("REPLAY_LOG"),
-  git_commit: git_commit
+  git_commit: git_commit,
+  data_set_directory: System.tmp_dir!() |> Path.join("datasets")
```

Comment:
`System.tmp_dir!()` is problematic here, because it's being evaluated only once, at compile-time, and you get back a random dir like `/var/folders/5g/2jxct52j2d5dlc9b621n69s40000gp/T/` which might go missing unless you recompile the app.

If we want to use `tmp_dir!` it will have to be evaluated at runtime. Or, we can hardcod something more permanent here.

Reply:
I _think_... I see you are doing `mkdir_p` later-on so maybe this is okay.